src/ario_event.lua

```diff
@@ -1,8 +1,9 @@
 local AOEvent = require("ao_event")
 local utils = require("utils")
 
+-- TODO: RENAME TO ARIOEvent
 -- Convenience factory function for prepopulating analytic and msg fields into AOEvents
-local function IOEvent(msg, initialData)
+local function ARIOEvent(msg, initialData)
```

Comment:
looks to be done

Reply:
Didn't rename the ioEvent variable on messages though.